### PR TITLE
Support synctex for Knitr files

### LINKED ISSUE #251: synctex does not work with knitr
If an Rnw file is build with the concordance option and the synctex.gz file is patched, then ctrl-alt-s should work for Rnw files.

Attached is a working example: [knitr_synctex.tar.gz](https://github.com/thomasjo/atom-latex/files/522508/knitr_synctex.tar.gz)

Please note that because atom-latex does not yet support the concordance option and patching, you will need to do this with the Makefile in the example.

To test this out:
```sh
# install R library patchSynctex
make rSnapshot

# compile pdf
make
```

Now try to use synctex in Atom; ctrl-alt-s in the Rnw file. It should take you to the location in pdf-view. It doesn't do anything.

If you add Rnw as a file extension to this function in composer.js
```javascript
  isTexFile (filePath) {
    // TODO: Improve will suffice for the time being.
    return !filePath || filePath.search(/\.(tex|lhs|Rnw)$/) > 0
  }
```
It works. But I'm not sure this is the best way to fix this.

### PR DESCRIPTION
**Pull request by harmsk**
Check for knitr files in sync command. resolves #251

### CODE REVIEW

File: lib/composer.js [hunk 1]
```diff
@@ -250,6 +250,11 @@ export default class Composer {
     return !filePath || filePath.search(/\.(tex|lhs)$/) > 0
```
Comment (thomasjo) [inline lib/composer.js]: Let's simply change this regex to
```js
/\.(tex|lhs|[rs]nw)$/i
```

File: lib/composer.js [hunk 2]
```diff
@@ -250,6 +250,11 @@ export default class Composer {
...
   }
 
+  isKnitrFile (filePath) {
+    // scope: text.tex.latex.knitr
+    return !filePath || filePath.search(/\.(Rnw|Snw|rnw|snw)$/) > 0
+  }
+
```
Comment (thomasjo) [inline lib/composer.js]: Remove this function. I realize you were told to add this, but I don't see why it's necessary.

**Comment by yitzchak** [comment]
@thomasjo I am on it. I'll make it happen.

**Comment by yitzchak** [comment]
Thanks @harmsk!

**Pull request merged**